src/main/java/org/wildfly/security/credential/source/CommandCredentialSource.java

```diff
@@ -88,10 +91,23 @@ public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential
                 throw new UndeclaredThrowableException(throwable);
             }
         }
-        try {
+        try (final InputStream errorStream = process.getErrorStream()) {
```

Comment:
A note that as of 9, ProcessBuilder.Redirect.DISCARD can be used to safely discard output.  But that simply translates into a file redirect to "NUL" on windows and "/dev/null" on other platforms, which is regarded to be safe and also prevents the requirement to start a thread (and possible weirdness in the event that the thread creation fails).

Reply:
The literal code for that in OpenJDK is simply:

```
        private static final File NULL_FILE = new File(
                (GetPropertyAction.privilegedGetProperty("os.name")
                        .startsWith("Windows") ? "NUL" : "/dev/null")
        );

```